managegroup.php

```diff
@@ -341,7 +341,13 @@
 	{
 		error($lang->group_no_members);
 	}*/
+	
 	$perpage = $mybb->settings['membersperpage'];
```

Comment:
I don't see any issues with negative numbers here, but i can convert all the checks to `< 1`.